scenarios/22_North_Pole.cfg

```diff
@@ -23,6 +23,85 @@
         [/lua]
     [/event]
 
+    #define IMPROVE_STATS_ON_FROZEN UNIT_FILTER
```

Comment:
#define in scenario should be #undef in same file

Reply:
preprocessor directives should not have whitespace before them

Reply:
Noted. Will fix.